snap/snapcraft.yaml

```diff
@@ -1,5 +1,5 @@
-name: couchdb
-version: 2.0
+name: couchdb-sklassen
```

Comment:
If you're signing up to support the snap package from here on out, leave this as `couchdb`, and we'll re-publish it in the snap store under the main couchdb account.

Reply:
In fact, even if you're not signing up for it, let's revert this to `couchdb`, if you end up publishing it yourself, you can maintain a fork with your own package name. Let's not have a personal name in the official repo for the snapfile. :wink: